### PR TITLE
Replace raw HTML with Twig for deploy policies

### LINKED ISSUE #493: Replace raw html with twig template
#### Observed Results:
There is raw html code in TLS policy

  
#### Expected behavior:

Replace this raw html for twig templates, using inclusion

Ref: [Include](https://twig.symfony.com/doc/2.x/tags/include.html)

### PR DESCRIPTION
**Pull request by Naylin15**
### Changes description
Create new twig files for the deploy policies
More changes to come

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [x] Docs have been added/updated

### References

Closes #493

### CODE REVIEW

**Comment by Naylin15** [comment]
I've been working on this, after a couple of issues the use of the twig template for the deploy application policy is working as it should
I'm still working with the template for the deploy file

**Comment by btry** [comment]
It looks good so far.

**Comment by Naylin15** [comment]
The deploy file doesn't have anymore raw html

**Comment by btry** [comment]
Nice ! If yo uextend this work to the 3 other deployment policies then we are done.

I'm unsure if we should modify to  the showValueInput in PolicyBase class (a single input tag ...)

**Comment by Naylin15** [comment]
@btry 

I'm not following you, I don't see any other policy that has raw html excepting the input you already mentioned in PolicyBase

**Comment by btry** [comment]
You're right ! so, time to review ?

**Comment by Naylin15** [comment]
Yes, this PR is ready!

File: inc/policydeployfile.class.php [hunk 1]
```diff
@@ -281,29 +281,37 @@ public function showValueInput($value = '', $itemType = '', $itemId = 0) {
...
-      return $out;
+
+      $removeDropdown = Dropdown::showYesNo('value[remove_on_delete]', $removeOnDelete, 
+                                                -1, ['display' => false]);
+
+      $data = [
+            'files'         => [
+                  'dropdown'        => $filesDropdown,
+                  'itemtype'        => $itemtype,
+                  'copy'            => $copyDropdown,
```
Comment (DIOHz0r) [inline inc/policydeployfile.class.php]: Please be more semantic and change the text copy and the var $copyDropdown to knownPath

**Comment by Naylin15** [comment]
I found an error when updating the policy

The update button is not being shown

captura de pantalla 2018-05-25 a la s 4 49 53 p m

How it should be:

captura de pantalla 2018-05-25 a la s 4 51 56 p m

**Comment by btry** [comment]
Hi, I fixed unit tests and CS issues.

**Comment by Naylin15** [comment]
Thank you @btry 

This PR is now ready

**Pull request merged**